src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  function sum(a) {
```

Comment:
The inner function is named as 'sum' while task description asks for 'getSum'. Consider renaming 'sum' to 'getSum' to meet the requirements.